psi4/src/read_options.cc

```diff
@@ -2741,6 +2746,19 @@ int read_options(const std::string &name, Options & options, bool suppress_print
     options.add_double("REG_PARAM",0.4);
     /*- tolerance for Cholesky decomposition of the ERI tensor -*/
     options.add_double("CHOLESKY_TOLERANCE",1.0e-4);
+    /*- Cutoff for occupation of MP2 virtual NOs in FNO-CCSD/CCSD(T).
+        Virtual NOs with occupations less than |dfocc__fno_tolerance|
+        will be discarded. This option is only used if |dfocc__fno| =
+        true. -*/
+    options.add_double("FNO_TOLERANCE", 1.0e-4);
+    /*- Cutoff for occupation of MP2 virtual NOs in FNO-CCSD/CCSD(T).
+        The number of virtual NOs is chosen so the occupation of the
+        truncated virtual space is |dfocc__fno_percentage| percent of
+        occupation of the original MP2 virtual space. This option is only
+        used if |fnocc__fno| = true. This keyword overrides
+        |fnocc__fno_tolerance|. -*/
+    //options.add_double("FNO_PERCENTAGE", 99.0);
+
```

Comment:
It would be very helpful if dfocc used exactly analogous keywords to fnocc to control FNO. That is, please implement  `OCC_TOLERANCE`, `OCC_PERCENTAGE`, `ACTIVE_NAT_ORBS`, and `NAT_ORBS` within dfocc module in read_options. The last one is optional, as I think it's mainly an internal routing kwd (users probably indicate via `energy('fno-ccsd')`. The first may then be referred to as `|dfocc__occ_tolerance|` in read_options docstrings.

Reply:
I have revised the option names as you discussed and I have added a test for fno-df-ccsd(t). @loriab

Reply:
Thanks for changing some of the names. But you're still referring to `|fnocc__` keywords in the dfocc section of read_options. And I don't think you actually implemented two of the three among `OCC_TOLERANCE`, `OCC_PERCENTAGE`, `ACTIVE_NAT_ORBS`, just added the text to read_options. Commented options are bad in read_options because the docs parser slurps them up anyways.

Reply:
I have revised the read_options.cc file and deleted commented options. @loriab

Reply:
Thanks for fixing up the names, @bozkaya. Do you think you'd have time to actually implement `OCC_PERCENTAGE` and `ACTIVE_NAT_ORBS` within the `dfocc` module? I think having multiple ways to set the FNO space is useful in the course of a FNO project.

Reply:
Hi Lori,

Thank you for your suggestions. I have implemented OCC_PERCENTAGE and ACTIVE_NAT_ORBS options. @loriab